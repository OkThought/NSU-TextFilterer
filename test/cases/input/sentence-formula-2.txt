There is a simple sentence. And there is a difficult one: a+b=c.