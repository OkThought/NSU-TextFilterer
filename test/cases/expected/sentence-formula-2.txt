There is a simple sentence.